### PR TITLE
Fix Infer Wikimedia Commons category from Wikidata to show photos 12303

### PR DESCRIPTION
**Pull request by Dima-1**

### CODE REVIEW

File: OsmAnd-java/src/main/java/net/osmand/wiki/WikiCoreHelper.java [hunk 1]
```diff
@@ -116,6 +133,9 @@ public static class Claims {
 		@SerializedName("P18")
 		@Expose
 		private final List<P18> p18 = null;
+		@SerializedName("P373")
+		@Expose
+		private final List<P18> p373 = null;
```
Comment (vshcherb) [inline OsmAnd-java/src/main/java/net/osmand/wiki/WikiCoreHelper.java:138]: Should be renamed to generic class ```P18 -> ?? ```

**Comment by vshcherb** [comment]
Also Obj-C will be needed, request is generally approved

File: OsmAnd-java/src/main/java/net/osmand/wiki/WikiCoreHelper.java (hunk 1 above)
Comment (ivanPyrohivskyi) [reply OsmAnd-java/src/main/java/net/osmand/wiki/WikiCoreHelper.java:138]: WikibaseProperty

**Comment by Chumva** [comment]
Looks like it is [fixed in iOS](https://github.com/osmandapp/OsmAnd-iOS/pull/3278), probably we have to sync code

**Pull request closed without merging**